packages/valory/skills/registration_abci/rounds.py

```diff
@@ -58,17 +59,23 @@ class RegistrationStartupRound(CollectSameUntilAllRound):
     payload_class = RegistrationPayload
     synchronized_data_class = BaseSynchronizedData
 
+    @property
+    def params(self) -> BaseParams:
+        """Return the params."""
+        return self.context.params
+
     def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
         """Process the end of the block."""
         if not self.collection_threshold_reached:
             return None
 
         try:
             _ = self.context.state.round_sequence.offence_status
+            # only use slashing if it is configured and the `use_slashing` is set to True
+            if self.params.use_slashing:
+                self.context.state.round_sequence.enable_slashing()
```

Comment:
Why inside the `try`?

Reply:
Just a style matter, otherwise i need this in after the except:
```
        except SlashingNotConfiguredError:
            self.context.logger.warning("Slashing has not been enabled!")
        else:
            if self.params.use_slashing:
                self.context.state.round_sequence.enable_slashing()
```